src/test/java/teammates/test/cases/ui/AllActionsAccessControlTest.java

```diff
@@ -73,7 +73,7 @@ private static void addUnregStudentToCourse1() throws Exception {
     }
 
     @Test
-    public void InstructorCoursesPage() throws Exception {
+    public void instructorCoursesPage() throws Exception {
```

Comment:
Chipping in a bit: this kind of instances might be better as `test*Page` instead?